AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRColorGradingAdded.py

```diff
@@ -146,42 +304,427 @@ def AtomEditorComponents_HDRColorGrading_AddedToEntity():
         # 7. Verify HDR Color Grading component is enabled.
         Report.result(Tests.hdr_color_grading_enabled, hdr_color_grading_component.is_enabled())
 
-        # 8. Enable the "Enable HDR Color Grading" parameter.
+        # 8. Toggle "Enable HDR Color Grading".
```

Comment:
nit: Toggle implies back and forth to me, but this only enables the parameter setting it True